package/MDAnalysis/analysis/align.py

```diff
@@ -380,6 +377,175 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
     return old_rmsd, new_rmsd
 
 
+class fit_trj(AnalysisBase):
```

Comment:
Please rename it to `AlignTrj` so that the naming convention is the same as the other alignment functions in this module. In PEP8 class names are written with *CamelCase*, functions and variables should be *snake_case*